### DISCUSSION TITLE
Filtering data providers

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8352] [Ideas]
Hey,

is there a way to filter rows like: "Does not start with" or "Does not contain"?

Would be cool!

**Comment by melohagan** [comment]
Hey @kasgithub8 

There is a 'Does not contain' for multi-select types:
<img width="576" alt="Screenshot 2022-10-21 at 19 20 55" src="https://user-images.githubusercontent.com/101575380/197263404-1140eae5-ea19-41fd-8129-30ccaf481b2f.png">

There's no 'does not start with' however. Feel free to raise feature requests for any specific filtering scenarios you would like!
